cleanlab/object_detection/dataset.py

```diff
@@ -0,0 +1,419 @@
+# Copyright (C) 2017-2023  Cleanlab Inc.
```

Comment:
would be nice if you could update one of the bottom parts of [this tutorial](https://github.com/cleanlab/cleanlab/blob/master/docs/source/tutorials/object_detection.ipynb) to showcase your plotting functions from the example outputs.ipynb script you linked in this PR.

Reply:
make sure the plots show class names though, not integer class IDs

Reply:
Sure! I can update the tutorial once I fix all the comments.